lib/src/test/kotlin/org/didcommx/didcomm/DIDCommForwardTest.kt

```diff
@@ -0,0 +1,260 @@
+package org.didcommx.didcomm
+
```

Comment:
Looks like this test is actually a Demo forward test. Makes sense to link to it from README (https://github.com/sicpa-dlab/didcomm-jvm#examples) where we link to demo scripts.

Reply:
added